webview_panels/src/modules/whatsNew/fonts.scss

```
@@ -0,0 +1,25 @@
+// Fonts for the What's New panel, inlined as base64 data URIs.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)